roles/openshift_cluster_metrics_hawkular/files/cluster-metrics/metrics.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: "v1"
+kind: "Template"
+metadata:
+  name: metrics-deployer-template
+  annotations:
+    description: "Template for deploying the required Metrics integration. Requires cluster-admin 'metrics-deployer' service account and 'metrics-deployer' secret."
+    tags: "infrastructure"
```

Comment:
We'll probably need some way to differentiate between the enterprise and origin metrics-deployer here. Maybe directly using the existing files instead of making a copy.

https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_examples/files/examples/v1.1/infrastructure-templates/enterprise/metrics-deployer.yaml

https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_examples/files/examples/v1.1/infrastructure-templates/origin/metrics-deployer.yaml